### ISSUE TITLE
Missing command line options.

### ISSUE THREAD

**Issue by severach** [atari800/atari800#29]
Years ago I was unable to get Linux Atari 800 emulators working. The reason is that the GUI fine for testing and small changes but doesn't work well for the extensive initial setup. The command line exposes more settings which always work once set up. Unfortunately I found that a few settings don't have command line options. Here is the sed line I used to fill in the missing settings. There may be more but these are the ones that I wanted.

````
  # No command line switches for some settings
  sed -e 's:^\(SDL_JOY_0_ENABLED\)=.*$:\1=0:g' \
      -e 's:^\(ATARI_FILES_DIR\)=.*$:\1=/home/chris/c/games/atari/romsdave:g' \
      -e 's:^\(SAVED_FILES_DIR\)=.*$:\1=/home/chris/c/games/atari:g' \
    -i ~/'.atari800.cfg'
````

**Comment by joysfera** [comment]
maybe you could explain your issue and fix of it in plain words. The `sed` command just disables first joystick for me. I doubt that an enabled joysticks would not allow you to run Atari 800 for years.

**Comment by severach** [comment]
The original issue is that it is nearly impossible to set up the emulator using the on screen configuration. After about an hour of config and testing, press the wrong button, and it all disappears. When you want to copy the config to another machine all you have is outdated junk.

This is irrelevant because the command line is the right way to set up the emulator. It still takes an hour to get all the settings right but it doesn't all disappear when you press the wrong button and once done you can copy it anywhere, adjusting a few settings to match the computer. Not only that, many of the needed settings are not found anywhere in the config UI. They are only available from the command line.

The current issue is that these settings do not have command line options. The only way I can set them is to start the emulator which writes `.atari800.cfg` then modify it then start the emulator again.

**Comment by ChoccyHobNob** [comment]
Which settings are you talking about? There are only a couple that cannot be set from the command line (1088K ram, stuff like that) which ones can only be set from the command line?

The most important setting for you is probably the /config setting which allows you to specify a config file to use

**Comment by severach** [comment]
Settings found in the GUI but missing from the command line:
````
SDL_JOY_0_ENABLED, default 1, 0 to disable joystick keyboard emulation
ATARI_FILES_DIR, the folder that Alt-R starts with.
SAVED_FILES_DIR, the folder that load and save start with.
````
SDL_JOY_0_ENABLED does not disable the joystick unless you have a keyboard based joystick.

Settings not in the GUI, only accessible from the command line:
````
-win-width
-win-height
-fs-width
-fs-height
````
The emulator doesn't have a resize helper so the only way to get the screen size perfect is by command line.

**Comment by joysfera** [comment]
The two FILES_DIR could be added to command line options.

Modifying the graphics resolution from within the GUI would be ideally done using some kind of a visual interactive way.

The SDL_JOY_0_ENABLED description seems weird, I'll double check what it is supposed to do (I'd think it is to override the keyboard joysticks but I'll need to review the implementation first).

**Comment by mikrosk** [comment]
Fixed by #77.
